nimble/host/include/host/ble_gap.h

```diff
@@ -610,6 +610,16 @@ struct ble_gap_event {
              * like to use.  The application callback should fill this in.  By
              * default, this struct contains the requested parameters (i.e.,
              * it is a copy of 'peer_params').
+             *
+             * For BLE_GAP_EVENT_L2CAP_UPDATE_REQ, only connection interval
+             * min/max and connection event length min/max can be changed.
```

Comment:
@zacwbond we are saying here that min/max CE might be changed but code does not copy those parameters from self_params.